src/AppComponents/Common/Reader/OsmMapReader.cpp

```diff
@@ -250,33 +347,37 @@ bool OsmMapReader::init(
 {
     using namespace Core::Common;
 
-    // Note: For further development in @ambrosys:os-matcher-amb#38
-    size_t const eachNth = 1;
+    // Simplify Track
+    // ToDo: With correct type handling and inheritance the boost::geometry::simplify can be used directly
+    typedef boost::geometry::model::d2::point_xy<double> xy;
+    float maxDistance = 1.0; // GPS granularity is 2m, allow half of it since it can be left and right
+    AppComponents::Common::Types::Track::PointList simplified{};
+    douglas_peucker_(pointList, simplified, maxDistance);
 
     double searchRadius;
     std::string pointsString;
     if (!useSingleSearchCircle_)
     {
         searchRadius_ = fetchCorridor_;
-        pointsString_ = Geometry::toWkt(Geometry::flattened_simple(pointList, eachNth));
+        pointsString_ = Geometry::toWkt(simplified);
     }
```

Comment:
Only here we need the simplified version, please move the code into this block.

Reply:
check pls